test/support/test_config.h

```diff
@@ -54,6 +54,8 @@
 #define _PSTL_STD_UNINITIALIZED_FILL_BROKEN (_MSC_VER == 1900)
 // GCC10 produces wrong answer calling exclusive_scan using vectorized polices
 #define TEST_GCC10_EXCLUSIVE_SCAN_BROKEN (_GLIBCXX_RELEASE == 10)
+// GCC7 std::get not returning const rvalue reference from const rvalue reference of tuple

```

Comment:
`not returning` - > `doesn't return`

Reply:
It was the text from original GCC issue at https://gcc.gnu.org/pipermail/gcc-help/2018-May/133865.html

Reply:
Fixed